app/forms/post_form.py

```diff
@@ -8,12 +8,10 @@
 
 class CreatePostForm(FlaskForm):
     DESCRIPTION_LENGTH_MAX = 2_200
-    ALLOWED_EXTENSIONS = set([ 'png', 'jpg', 'jpeg', 'gif'])
-
 
     post_image = FileField(
         'Image',
-        render_kw={'style': 'height: auto', 'onchange':'previewImage(event)', 'accept': 'image/*'},
```

Comment:
Don't we want to limit the type of files displayed in the pop up file explorer to only our accepted image types?

Reply:
sorry it was me who deleted that, ill add it back

Reply:
Still not fixed

Reply:
fixed